convex/boards.ts

```
@@ -0,0 +1,280 @@
+    userId: v.id("users"),
+    cursor: v.optional(v.id("boards")),
+    limit: v.optional(v.number()),
+  },
+  handler: async (ctx, args) => {
+    const user = await ctx.db.get(args.userId);
+    if (!user) {
+      throw new Error("User not found");
+    }
+
+    const limit = args.limit ?? 10;
+    const queryArgs: any = {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)